route/emitter_test.go

```diff
@@ -138,10 +169,20 @@ var _ = Describe("Emitter", func() {
 	})
 
 	Context("When the publisher returns an error", func() {
+		useIngress = true
+
 		BeforeEach(func() {
 			publisher.PublishReturns(errors.New("Failed to publish message"))
 		})
 
 		assertInteractionsWithFakes()
 	})
+
+	Context("When not using an ingress", func() {
+		useIngress = false
+
+		BeforeEach(func() {
+			publisher.PublishReturns(errors.New("Failed to publish message"))
+		})
+	})
```

Comment:
This Context doesn't seem to test anything. Our guess is that you forgot to call `assertInteractionsWithFakes` 🙂 

cc @mnitchev